### PR TITLE
Add/update background examples

### LINKED ISSUE #562: CSS examples: example needed for background
https://developer.mozilla.org/en-US/docs/Web/CSS/background

This is quite a complex one I think. It might be worth looking at some of the related examples we've already done, like: https://developer.mozilla.org/en-US/docs/Web/CSS/background-image

It would also be worth writing examples for the other background-* properties at the same time, as they could share a lot of the design.

### PR DESCRIPTION
**Pull request by mfluehr**
Fixes #562. This adds all the `background-*` properties that are not yet created. It also makes various updates to some existing ones.

The moon image is taken from https://commons.wikimedia.org/wiki/File:Moon.jpg, which is in the public domain.

### CODE REVIEW

File: live-examples/css-examples/backgrounds-and-borders/background-position-x.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
 </section>
 
 <div id="output" class="output large hidden">
-    <section id="default-example" class="display-block">
+    <section id="default-example">
```
Comment (wbamberg) [inline live-examples/css-examples/backgrounds-and-borders/background-position-x.html]: This example, and the other two background-position examples, don't seem to work for me any more: I don't see the star or the background colour. It seems to be the removal of `class="display-block"` that's done this. Do you see that as well?

File: live-examples/css-examples/backgrounds-and-borders/background-attachment.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+</div>
+</section>
+
+<div id="output" class="output hidden">
+    <section id="default-example">
+          <div id="example-element">
+London. Michaelmas term lately over, and the Lord Chancellor sitting in Lincoln's Inn Hall. Implacable November weather. As much mud in the streets as if the waters had but newly retired from the face of the earth, and it would not be wonderful to meet a Megalosaurus, forty feet long or so, waddling like an elephantine lizard up Holborn Hill.
+          </div>
+    </section>
+</div>
```
Comment (wbamberg) [inline live-examples/css-examples/backgrounds-and-borders/background-attachment.html]: This is a really nice example :).

File: live-examples/css-examples/backgrounds-and-borders/background.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+</button>
+</div>
+
+<div class="example-choice" initial-choice="true">
+<pre><code class="language-css">background: center / contain no-repeat url("../../media/examples/firefox-logo.svg"),
+            #cf4 35% url("../../media/examples/lizard.png");</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
```
Comment (wbamberg) [inline live-examples/css-examples/backgrounds-and-borders/background.html]: This one is a bit...lively. I wonder if we should tone down the colours a bit. But that's just a subjective opinion.

File: live-examples/css-examples/backgrounds-and-borders/background-position-x.html (hunk 1 above)
Comment (mfluehr) [reply live-examples/css-examples/backgrounds-and-borders/background-position-x.html]: Yes, sorry about that.

**Pull request merged**